server/grpcservice/service.go

```diff
@@ -226,27 +226,17 @@ func (s *Service) GetTables(ctx context.Context, req *metaservicepb.GetTablesReq
 
 	tableMap := make(map[uint32]*metaservicepb.ShardTables, len(tables))
 	for shardID, shardTables := range tables {
+		tableMap[shardID] = &metaservicepb.ShardTables{
```

Comment:
`tableMap` sounds verbose, and maybe we should call it `tablesPb`. As for `TablesMap` in the proto file we should rename it to `ShardTables` in another pr.